Source/Applications/SystemCenterNotification/Scripts/TSX/Subscriptions/Event/ByAllSubscription.tsx

```diff
@@ -170,8 +185,10 @@ const ByAllSubscription = (props: IProps) => {
                     />
                 </div>
             </div>
-            <Warning Show={showWarning} Title={'Approve Norification Subscriptions'} Message={`This will approve all ${nApproval} Subscriptions that are currently pending`}
-                CallBack={(c) => { setShowWarning(false); if (c) approveAll(); }} />
+            <Warning Show={showApproveWarning} Title={'Approve Norification Subscriptions'} Message={`This will approve all ${nApproval} Subscriptions that are currently pending`}
```

Comment:
```suggestion
            <Warning Show={showApproveWarning} Title={'Approve Notification Subscriptions'} Message={`This will approve all ${nApproval} Subscriptions that are currently pending.`}
```